components/script/dom/url.rs

```diff
@@ -24,19 +26,24 @@ pub struct URL {
 
     // https://url.spec.whatwg.org/#concept-urlutils-get-the-base
     base: Option<Url>,
+
+    // https://url.spec.whatwg.org/#dom-url-searchparams
+    query: JS<URLSearchParams>,
```

Comment:
Chrome doesn't seem to support searchParams at all? It autocompletes in the console but is undefined.